.github/workflows/acquia_cms_ci.workflow.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           node-version: 12.13.1
       - uses: shivammathur/setup-php@v2
         with:
-          php-version: 8.1
+          php-version: 8.2
```

Comment:
would be better if CI workflow run on both PHP 8.1 and PHP 8.2

Reply:
@apathak18 Yes, I'll create a separate ticket to run CI on PHP 7.4/8.0/8.1 as well.

Reply:
+1